steps/pytorch/models.py

```diff
@@ -89,6 +89,12 @@ def _fit_loop(self, data):
         outputs_batch = self.model(X)
         partial_batch_losses = {}
 
+        assert len(targets_tensors) == len(self.loss_function) == len(outputs_batch),\
+            'Number of targets, model outputs and elements of loss function must equal. '\
+            'You have n_targets=' + str(len(targets_tensors)) + ', n_model_outputs=' + str(len(outputs_batch)) + \
```

Comment:
@kamil-kaczmarek I would recommend using """part of message {} rest of message""".format(values) rather than + str(values) . it lets you reuse the same values with the concept of "{0} {1} {0}".format(value0,value1) among other usefull things